coreTable/OwidTable.test.ts

```diff
@@ -451,3 +479,62 @@ describe("time domain", () => {
         expect(table.get("gdp")!.uniqTimesAsc).toEqual([1950, 1970, 2000])
     })
 })
+
+describe("tolerance", () => {
+    it("can apply tolerance to a column", () => {
+        const table = OwidTable.fromDelimited(
+            `gdp,year,entityName,entityId,entityCode
+,2000,usa,1,
+0,2001,usa,1,
+,2002,usa,1,
+,2003,usa,1,
+2,2004,uk,2,
+1,2005,usa,1,`,
+            [
+                { slug: "gdp", type: ColumnTypeNames.Numeric },
+                { slug: "year", type: ColumnTypeNames.Year },
+            ]
+        )
+
+        const toleranceTable = table.fillColumnWithTolerance("gdp", 1)
+
+        expect(toleranceTable.rows).toEqual(
```

Comment:
this is some next level jesting :)

Reply:
Yeah I scrolled through some stackoverflow. :) 

We should maybe add some Jest utils, just mentioned it on Slack.